app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -164,6 +165,7 @@ def queue_signal(*args, deliver_on: nil)
 
   def temp_configuration_script
     ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript.new(
+      :name        => playbook.name,
```

Comment:
@lfu Where is the name going to get used?

Reply:
That name will be used for ```ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job```.

<img width="999" alt="Screen Shot 2019-06-21 at 15 31 42" src="https://user-images.githubusercontent.com/2593270/59946866-c3c49d80-9439-11e9-9c7a-e01da2459573.png">

Reply:
👍